gis_exercise_2.py

```diff
@@ -121,8 +121,12 @@
 from shapely.geometry import Polygon, Point, LineString
 
 # x = lon, y = lat
-for index, rows in kruger_park.iterrows():
-    kruger_park.loc[index, 'geometry'] = Point((kruger_park.loc[index, 'lon']), (kruger_park.loc[index, 'lat']))
+# This is WAY too slow. 
+#for index, rows in kruger_park.iterrows():
+#    kruger_park.loc[index, 'geometry'] = Point((kruger_park.loc[index, 'lon']), (kruger_park.loc[index, 'lat']))
+
+# this is a faster way of doing it. Not sure if it's needed. I'll review that later.
+kruger_park['geometry'] = [Point(x, y) for x,y in kruger_park[['lat', 'lon']].values]
```

Comment:
Man, I feel really stupid...

I have been following the course of 2018 and I just noticed today that there is also [one for 2019](https://automating-gis-processes.github.io/site/lessons/L2/exercise-2.html) available.
The one from 2019 explains so much of what I was pondering about and what I did wrong :D

oh well. better late than never I guess haha